src/goose/cli/main.py

```diff
@@ -97,6 +97,43 @@ def list_toolkits() -> None:
         print(f" - [bold]{toolkit_name}[/bold]: {first_line_of_doc}")
 
 
+@goose_cli.group()
+def providers() -> None:
+    """Manage providers"""
+    pass
+
+
+@providers.command(name="list")
+def list_providers() -> None:
+    providers = load_plugins(group="exchange.provider")
+
+    def extract_envs_from_doc(lines: List[str]) -> List[str]:
+        required_envs = []
+        env_block = False
+        for line in lines:
+            if "Required env vars:" in line:
+                env_block = True
+                continue
+            if env_block:
+                if line.strip():
+                    required_envs.append(line.strip())
+                else:
+                    break
+        return required_envs
+
+    for provider_name, provider in providers.items():
+        lines_doc = provider.__doc__.split("\n")
+        first_line_of_doc = lines_doc[0]
+        env_lines = extract_envs_from_doc(lines_doc)
+
+        print(f" - [bold]{provider_name}[/bold]: {first_line_of_doc}")
+        if env_lines:
+            env_required = ", ".join(env_lines)
+            print(f" - [dim]env vars: {env_required}")
```

Comment:
```suggestion
            print(f"        [dim]env vars: {env_required}")
```

Reply:
i think itll be a bit easier to read without the dash and with more indentation

Reply:
Instead of 

```
print(f" [dim]env vars: {env_required}")
```

can we do 

```
print(f" [dim]env vars required: {env_required}")
```

When I looked at the screen shot for the first time it was not immediately clear if it was listing out the env vars the user has defined in their environment or if those were the required env vars

Reply:
yeah, I think I like `env vars required` more too